kahuna/public/stylesheets/main.css

```diff
@@ -1477,15 +1526,16 @@ FIXME: what to do with touch devices
 }
 
 .image-crop {
-    border: solid 5px transparent;
-    max-width: 140px;
-    margin-right: 5px;
+    margin-bottom: 10px;
+    min-width: 100%;
     box-sizing: border-box;
+    background-color: #333;
 }
+
 .image-crop--selected,
 .result--selected,
 .result--selected:hover {
-    border-color: #00adee;
+    border: solid 2px #00adee;
```

Comment:
Are we aware on the effect this `2px` border has on `result--selected`?

![border](https://cloud.githubusercontent.com/assets/836140/9533558/c21dcb52-4d0a-11e5-8d04-19b6160c854a.gif)

Pre selection, `result` has a [transparent 5px border](https://github.com/guardian/grid/blob/master/kahuna/public/stylesheets/main.css#L718), decreasing it to 2px makes it look a little jumpy.